admin/adminActions.php

```diff
@@ -1248,6 +1253,28 @@ public function generateRegistrationLink(array $params)
 		return "Please give the user the following link: <br>".$thisURL.'?emailToken='.$emailToken;
 	}
 
+	public function changeReliability(array $params)
+	{
+		global $DB;
+
+		if (!isset($params['gameID']))
+		{
+			return "Please enter a game ID.";
+		}
+		
+		if (!isset($params['newRating']) || (!preg_match('/^[0-9][0-9]?$|^100$/', $params['newRating'])))

```

Comment:
Can you run another query that gets the min(rr) from the users in the game and display's an error message if they're trying to make the RR higher than what someone already in the game has?

Reply:
I was thinking of not doing this because as is a player can play in a game where the min RR is higher than their rating so long as they joined the game before their RR dropped below the minimum. Similarly, it makes sense that a player would be still able to play in a game if the RR requirement was changed if they were already in the game.

Reply:
okay